openedx/core/djangoapps/demographics/docs/decisions/0001-demographics-djangoapp-api.rst

```diff
@@ -0,0 +1,22 @@
+Django Application to Support Demographics Features
+---------------------------------------------------
+
+
+Status
+======
+
+Accepted
+
+Context
+=======
+
+To support demographics features and the IDA we need to be able to access the
+current state of a User from the LMS (i.e. Program Enrollments, Enterprise status).
```

Comment:
Hi @staubina -- could you clarify why this was added to the core instead of added as a plugin?